packages/preview/modern-ysu-thesis/0.1.0/template/thesis.typ

```diff
@@ -142,10 +142,8 @@ asdasd
   ) <timing-tlt>
 ]))
 
-#figure(
-  image("..\assets\vi\ysulogo.png", width: 20%),
-  caption: [图片测试],
-) <ysu-logo>
+#let logo = image.with("assets/vi/ysulogo.png")
```

Comment:
This needs to be defined in the package (e.g. `lib.typ`) and imported from there.

This is because here, it will be resolved relative to the user's working directory / the project root whereas in the package, resolution is relative to the package root. Since the file is in the package, only the latter will work.

Reply:
ok,i have remove the picture